Shopipy.ProductManagement/DependencyInjection.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Shopipy.Shared.Services;
+using Shopipy.ProductManagement.Services;
+
+namespace Shopipy.ProductManagement
+{
+    public static class DependencyInjection
+    {
+        public static IServiceCollection AddCategoryManagement(this IServiceCollection categories)
```

Comment:
Probably forgot to change, but should be AddProductManagement

Reply:
Also in ApiService program.cs you are not adding product management, how were you even able to call your Http requests without getting DI errors?

Reply:
done